EnergySage/es-ds
# feat: update marketplace report link

### 🔗 Linked issue



[CED-1867: Publishing Marketplace Report no. 19](https://energysage.atlassian.net/browse/CED-1867)

### ❓ Type of change




### 📚 Description





- Update link to Marketplace Intel Report in navbar
- The associated images are stored in `es-site` and they were updated using the same filenames

### 🥼 Testing



- Test locally at http://localhost:8500/organisms/nav-bar
- Hover over the About Us menu
- Click the link for "New Research / EnergySage Intel's latest Solar & Storage Marketplace Report"
- It should take you to the new press release (hardcoded to the live site) at https://www.energysage.com/press/energysage-marketplace-intel-report-19/

#### 🧐 Feedback Requested / Focus Areas



-

### 📝 Checklist





- [x] I have documented testing approach
